### PR TITLE
Added void cloak hood

### PR DESCRIPTION
**Pull request by NotSoDana**
## About the PR

A hood for void cloak

## Why / Balance

Void cloak has something like hood on back, but don't have it for real. So, why not to add it?

## Technical details

yaml

## Media

image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Dezzzix
- add: Now you can wear a hood in void cloak

### CODE REVIEW

**Comment by beck-thompson** [comment]
I'd add that void typing indicator as well!

**Comment by UbaserB** [comment]
The shading doesn't really match up with the cloak.

**Comment by NotSoDana** [comment]
k, i'll resprite it

**Comment by UbaserB** [comment]
> k, i'll resprite it

Are you coming back to this PR?

**Comment by NotSoDana** [comment]
> 
> Are you coming back to this PR?

Yep, reshaded

File: Resources/Prototypes/Entities/Clothing/Head/hoods.yml [hunk 1]
```diff
@@ -745,3 +745,20 @@
       - state: coatybits-equipped-HELMET
         color: "#EBE216"
       - state: winterbits-equipped-HELMET
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatHoodVoidCloak
+  name: void cloak hood
+  description: A hood of a void cloak. For those who have gone to the dark side of the force..
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Clothing/Head/hoods.yml:753]: the hood of a void cloak, not a hood of the void cloak

**Comment by Everturning** [comment]
can you make it so the sprite for the hood on the actual cloak is removed when the hood is up?

**Comment by lzk228** [comment]
> can you make it so the sprite for the hood on the actual cloak is removed when the hood is up?

wyci

**Comment by NotSoDana** [comment]
> can you make it so the sprite for the hood on the actual cloak is removed when the hood is up?

gigawyci

**Pull request merged**